cmd/world/root/root.go

```diff
@@ -24,10 +25,7 @@ func init() {
 
 	// Register subcommands
 	rootCmd.AddCommand(cardinal.BaseCmd)
-
 	rootCmd.AddCommand(evm.EVMCmds())
```

Comment:
Why was config.AddConfigFlag(rootCmd) removed?

Reply:
My bad. looks like i accidentally deleted it.
I've added it again as before.